### ISSUE TITLE
_flanking variable undefined

### ISSUE THREAD

**Issue by xetra11** [genesis92x/VcomAI-3.0#100]
**Describe the bug**
I just saw a error appearing in the *.rpt file

**To Reproduce**
Let a combat situation happen

**Expected behavior**
No error message about a non defined variable `_flanking`

**Screenshots**

**VCOM Version:**
 - Version v3.3.0
 
**Additional mods used:**
 - e.g CBA_A3
- ALiVE
- CoopR
- ACRE2
- BWMod
- ACE
- many model mods

**Additional context**
```12:20:02 Error in expression <en
{
_situation = "FLANKING";
};
[_grp, _flanking] call VCM_fnc_SetSituation; 

>
12:20:02   Error position: <_flanking] call VCM_fnc_SetSituation; 

>
12:20:02   Error Undefined variable in expression: _flanking
12:20:02 File \vcomai\Vcom\Functions\VCM_Functions\garrison\fn_UnGarrisonL.sqf [VCM_fnc_UnGarrisonL], line 21```

**Comment by Freddo3000** [comment]
Fixed with 11855d88b1f1608d2de405ab6bea885e7512a61a

**Comment by xetra11** [comment]
Will QA when in next release
